### ISSUE TITLE
No BindingDB_All.tsv exists.

### ISSUE THREAD

**Issue by CallMeDek** [Shen-Lab/DeepAffinity#8]
Hi,

Thank you for your work.
I am following your work with "DeepAffinity/data/script/split_data_script/README.md"
But there is a problem. 
According to README.md, there must be BindingDB_All.tsv in google drive(https://drive.google.com/open?id=1_uZVTBVPeeF64joitPU8uDZnfIGS1me0).
But there's no tsv file exist instead only BindingDB_ALL_2D.sdf is available.
So, I can't proceed anymore. 

Thank you
Dek

**Comment by Shen-Lab** [comment]
Hi @CallMeDek, Thank you for your interest in DeepAffinity.  You are right in finding the link wrong for the file needed, BindingDB_All.tsv, for firstStep.py.  Instead the file can be downloaded from bindingdb.org.  Simply proceeds to Download and find the corresponding file in the latest version. For instance, the latest as of today is the 2022-09-01 version called "BindingDB_All_2022m8.tsv.zip". Unfortunately we cannot locate the version we used "BindingDB_All_2018m8.tsv.zip" at this moment.  If that version is critical, I am reaching @AstroSign and I have also contacted BindingDB, but there is no guarantee. Our apologies for any inconvenience this may cause to you.  

Thanks again,
DeepAffinity Team
